rest_framework_json_api/utils.py

```diff
@@ -186,6 +186,14 @@ def get_related_resource_type(relation):
         relation_model = relation.model
     elif hasattr(relation, 'get_queryset') and relation.get_queryset() is not None:
         relation_model = relation.get_queryset().model
+    elif (
+            getattr(relation, 'many', False) and
+            hasattr(relation.child, 'Meta') and
+            hasattr(relation.child.Meta, 'model')):
+        # For ManyToMany relationships, get the model from the child
+        # serializer of the list serializer
+        # TODO Test coverage
```

Comment:
Could you either add a test to cover this line or delete the `TODO` comment? A test would be preferable, but understand if you can't. I don't think that `TODO` comments should be allowed in a master branch. Experience has showed me that they almost always languish and are forgotten until someone comes by later and deletes the comment.